src/display/annotation_layer.js

```diff
@@ -976,6 +1016,15 @@ class CheckboxWidgetAnnotationElement extends WidgetAnnotationElement {
       );
     }
 
+    this._setBorderAndBackgroundColors(element);
+
+    const { fontColor } = this.data.defaultAppearanceData;
```

Comment:
Why is this necessary only here and not elsewhere? Moreover, checkbox widgets don't contain text AFAIK, so what does this do?

Reply:
Checkboxes contains text for the `on` state: most of the time one char using `ZapfDingbats` font.